code/modules/projectiles/ammunition.dm

```diff
@@ -50,7 +50,7 @@
 	return
 
 /obj/item/ammo_casing/attackby(obj/item/I as obj, mob/user as mob, params)
-	if(istype(I, /obj/item/ammo_box))
+	if(istype(I, /obj/item/ammo_box) && !istype(I, /obj/item/ammo_box/magazine/wt550m9))
```

Comment:
This check is sort of snowflakey and the reason it's here is not that evident at first glance. Instead I think we can make it more generic with a new `slow_loading` var on magazines and do something like this:
```dm
/obj/item/ammo_casing/attackby(obj/item/I as obj, mob/user as mob, params)
	if(istype(I, /obj/item/ammo_box/magazine)
		var/obj/item/ammo_box_magazine/mag = I
		if(mag.slow_loading)
			return
	...
```

Reply:
Yeah that actually makes a lot of sense, I know some people have been interested in using a similar reload system for other guns so may as well plan for that by making it a var instead of a snowflake check.